### PR TITLE
feat(Catalog Management): re-gen service after recent API changes

### PR DESCRIPTION
**Pull request by benbuchanan**
## PR summary
Re-generating Go SDK after API updates

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Current vs new behavior  
New features associated with API updates

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by benbuchanan** [comment]
Examples tests:
```
Ran 599 of 627 Specs in 37.819 seconds
SUCCESS! -- 599 Passed | 0 Failed | 0 Pending | 28 Skipped
PASS
ok  	github.com/IBM/platform-services-go-sdk/catalogmanagementv1	38.914s
```

Integration tests:
```
Ran 610 of 666 Specs in 38.160 seconds
SUCCESS! -- 610 Passed | 0 Failed | 0 Pending | 56 Skipped
PASS
ok  	github.com/IBM/platform-services-go-sdk/catalogmanagementv1	38.933s
```

File: catalogmanagementv1/catalog_management_v1_examples_test.go [hunk 1]
```diff
@@ -499,6 +499,125 @@ var _ = Describe(`CatalogManagementV1 Examples Tests`, func() {
...
+			b, _ := json.MarshalIndent(accessListBulkResponse, "", "  ")
+			fmt.Println(string(b))
+
+			// end-delete_share_approval_list
+
+			Expect(err).To(BeNil())
+			Expect(response.StatusCode).To(Equal(200))
+			Expect(accessListBulkResponse).ToNot(BeNil())
+		})
+
```
Comment (padamstx) [inline catalogmanagementv1/catalog_management_v1_examples_test.go:620]: There doesn't appear to be a code block for the GetOfferingSourceArchive() operation.

**Review by padamstx — COMMENTED** [review]
I noticed that there's no example code block for the GetOfferingSourceArchive operation.  But otherwise, this looks good.

**Review by padamstx — APPROVED** [review]
LGTM

**Pull request merged**